deployment/provision.sh

```diff
@@ -64,6 +64,12 @@ wait_till_kubernetes_created ${w_eu_group} ${ACS_NAME}
 wait_till_kubernetes_created ${jenkins_group} ${ACS_NAME}
 [[ $? -ne 0 ]] && return 1
 
+if [[ -n "$MS_CORP" ]]; then
```

Comment:
Please update the notes of running `source provision.sh` to reflect this change, saying internal users need to set this variable.

Reply:
Updated.